lib/rails_multitenant/version.rb

```diff
@@ -1,3 +1,3 @@
 module RailsMultitenant
-  VERSION = "0.1.0"
+  VERSION = '0.2.0'
```

Comment:
We should consider applying SemVer to our versions. The rename of `multitenant_model_on` was a compatibility breaking change.